github.js

```diff
@@ -40,6 +41,15 @@ exports.getOpenBacklogIssues = function() {
    }).then(getAllPages);
 };
 
+exports.getIssueEvents = function(issue) {
+   debug("Getting events for issue %s", issue.getNumber());
+   return getIssueEvents({
```

Comment:
Is there a reason we're not using `includeDefaultParams` here?

Reply:
Yeah, the issue number param is `issue_number` instead of just `number` like all the others. :-(